README.md

```diff
@@ -1 +1,13 @@
 # Bunnymand.github.io
+
+## What should I do with my project
+
+[Rescue](http://www.bunnymen.com/)
+
+Can i do **bold**
+
+[And add a picture of the Bunnymen](http://www.bunnymen.com/wp-content/uploads/2012/04/bunnymen-mail.jpg)
```

Comment:
You need an excalmation mark
[!text](http://i100.independent.co.uk/image/25946-1xzimwm.png)